epub33/core/index.html

```diff
@@ -667,6 +673,47 @@ <h3>Conformance Criteria</h3>
 						<a href="#sec-publication-resources"></a>.</p>
 
 				<p>The rest of this specification covers specific conformance details.</p>
+
+				<section id="sec-conformance-validation">
+					<h4>Conformance Checking</h4>
+
+					<p>Due to the complexity of this specification and number of technologies used in <a>EPUB
+							Publications</a>, <a>EPUB Creators</a> are advised to use an <a>EPUB Validator</a> to check
+						the conformance of their content prior to distributing it. Finding and fixing errors will help
+						improve the interoperability of EPUB Publications across <a>Reading Systems</a>.</p>
+
+					<p>An EPUB Validator MUST be capable of checking the machine-processable normative requirements of
+						this specification and reporting on their presence. As these types of issues will generally
+						result in EPUB Publications not rendering, or rendering in inconsistent ways, it is RECOMMENDED
+						that EPUB Validators report them at the highest severity (e.g., as errors and/or critical
+						errors).</p>
+
+					<div class="note">
+						<p>It will not be possible for an EPUB Validator to check all the requirements as some issues
+							may require human inspection, some may not be checkable until <a>EPUB Content Documents</a>
+							are rendered (i.e., due to scripting), etc.</p>
+					</div>
+
+					<p>Although an EPUB Validator SHOULD also check the machine-processable normative recommendations of
```

Comment:
I presume this is a cut-and-paste issue; this section talks about non-normative features, right?

```suggestion
					<p>Although an EPUB Validator SHOULD also check the machine-processable non-normative recommendations of
```

Reply:
Isn't it:

- normative requirements = MUST/REQUIRED
- normative recommendations = SHOULD/RECOMMENDED

Non-normative recommendations are best pratices in notes and such, no?

(I don't know what to make of MAY/OPTIONAL, but I'm not sure we need to say anything about those for validation.)